### ISSUE TITLE
Power lines

### ISSUE THREAD

**Issue by gtrus21** [osmandapp/OsmAnd#14536]
Прошу добавить в раздел "скрыть" пункт "ЛЭП" (линии электропередач: power=line, power=minor_line, power=tower и power=pole)

**Comment by tradavyvy** [comment]
I take it this is a request for OPTIONAL hiding of powerline features. Because as features in the landscape they are sometimes quite handy for orientation purposes.

**Comment by letypequividelespoubelles** [comment]
Yes, for me it would be a regretion if it's not render anymore. Very usefull for orientation and free flying.

**Comment by gtrus21** [comment]
Yes, it's optional
